toolkit/activity.py

```diff
@@ -206,7 +210,7 @@ def read_file(self, filepath):
         if self.__state == _NEW_INSTANCE:
             self.__state = _PRE_INSTANCE
         elif self.__state == _PRE_INSTANCE:
-            self.__instance();
+            self.__instance()
```

Comment:
Agreed.  I liked seeing evidence of C programmers.  :grinning:

Reply:
:)